lms/services/email_preferences.py

```diff
@@ -112,10 +142,18 @@ def _url(self, route, h_userid, tag):
             _query={"token": self._encode_token(TokenPayload(h_userid, tag))},
         )
 
-    def instructor_digest_unsubscribe(self, h_userid) -> None:
-        """Unsubscribe `h_userid` from emails of type `tag`."""
+    def unsubscribe(self, h_userid, tag) -> None:
+        """Unsubscribe `h_userid` from instructor digest emails."""
         email_preferences = self.get_preferences(h_userid)
-        email_preferences.unsubscribe_instructor_digest()
+        if tag == EmailTypes.INSTRUCTOR_DIGEST:
+            email_preferences.mon = email_preferences.tue = email_preferences.wed = (
+                email_preferences.thu
+            ) = email_preferences.fri = email_preferences.sat = (
+                email_preferences.sun
+            ) = False
```

Comment:
Is this a way to set all values to `False`? If so, I would maybe set them individually in consecutive lines, as I think this is a bit confusing to read.

Reply:
This gets very weird with the formatting. I'll change it.